src/components/PreviewCard/LinkPreviewCard.tsx

```diff
@@ -0,0 +1,29 @@
+import Head from 'next/head';
+
+interface LinkPreviewCardProps {
+    taskDetails: any;
+    taskURL: any;
```

Comment:
Isn't taskUrl a string?
If we know the type could we update it instead of any?

Reply:
Yes @endeavourmonk has to fix it, we can't push more "any" type in the codebase.